spec/models/leitner_method_spec.rb

```diff
@@ -47,84 +47,35 @@
       it { expect(card.wrong_counter).to eq 0 }
       it { expect(card.review_date).to eq DateTime.now + 1.month }
     end
+
+    context "field correct_counter is not equal 6" do
+      let(:card) { FactoryGirl.create :card, correct_counter: 5, review_date: DateTime.now }
```

Comment:
Line is too long. [92/80]<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.